pkg/queue/concurrency_state.go

```diff
@@ -22,6 +22,8 @@ import (
 	"go.uber.org/zap"
 )
 
+const ConcurrencyStateTokenVolumeMountPath = "/var/run/secrets/tokens"
```

Comment:
you can tell gosec to ignore this with a comment like the one in https://github.com/knative/serving/blob/d5d489c0babd5c9ae9a37d707c06df92d70711f5/vendor/knative.dev/pkg/kmeta/names.go#L45.